esx_service/cli/vmdkops_admin.py

```diff
@@ -865,10 +880,19 @@ def tenant_vm_rm(args):
     error_info = auth_api._tenant_vm_rm(args.name, args.vm_list)
 
     if error_info:
-        return operation_fail(error_info)
+        return operation_fail(error_info.msg)
     else:
         print("tenant vm rm succeeded")
 
+def tenant_vm_replace(args):
+    """ Handle tenant vm replace command """
+    error_info = auth_api._tenant_vm_replace(args.name, args.vm_list)
+
+    if error_info:
+        return operation_fail(error_info.msg)
+    else:
```

Comment:
Yes, but I looked at handler for other AdminCLI command, they seem to follow this if-else pattern. So I prefer to leave it as it is.